app/javascript/react/components/ShopShowPage.js

```diff
@@ -1,7 +1,19 @@
 import React, {useState, useEffect} from "react"
+import { Link } from "react-router-dom"
+import ShopReviewsList from "./ShopReviewsList"
 
 const ShopShowPage = (props) => {
-  const [shop, setShop] = useState({})
+  const [shop, setShop] = useState({
+    name: "",
+    address: "",
+    city: "",
+    state: "",
+    zip: "",
+    telephone: "",
+    website: "",
+    picture: "",
+    reviews: []
```

Comment:
the only things actually needed here for default state is the `reviews` array. listing all properties is fine, but adds overhead as the app grows and things are added/removed from a shop, you have to remember to change this state